pull_request_template.md

```diff
@@ -0,0 +1,39 @@
+## :pushpin: Summary
+
+<!-- If merged, this PR.... 
+This should be a short TL;DR that includes the purpose of the PR.
+-->
+### :speech_balloon: 
+Please consider using [conventional comments](https://conventionalcomments.org/) when reviewing this PR.
+
+<!--
+Examples: 
```

Comment:
opening my first PR with this and having these be comments doesn't seem helpful since they are for the reviewer, not the person who is opening the PR. The person reviewing the PR won't see these. Should we move these into the actual text of the PR template? @MelSumner wdyt?